pkg/certificateprovider/azurekeyvault/provider.go

```diff
@@ -79,11 +79,6 @@ func (s *akvCertProvider) GetCertificates(ctx context.Context, attrib map[string
 		return nil, nil, re.ErrorCodeConfigInvalid.NewError(re.CertProvider, providerName, re.AKVLink, nil, "clientID is not set", re.HideStackTrace)
 	}
 
-	azureCloudEnv, err := parseAzureEnvironment(cloudName)
```

Comment:
do we not need cloud env information now? Is this because the new SDK can infer cloud type based on the URI?

Reply:
I couldn't find any usecases for the couldName variable, other than extracting azureenv which is removed here because the new SDK only needs the vault's URI when creating a client. So I removed it.